app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,10 @@
 class PagesController < ApplicationController
   def home
   end
+  
+  def menu
```

Comment:
I think this might make more sense in a `MenusController` so you can group together actions that are about menus.

Reply:
That makes sense